src/BucketBrigade.qs

```diff
@@ -94,7 +94,7 @@
     /// as a one-hot encoding.
     /// ## targetRegister
     /// The register that will have the memory value transferred to.
-    internal operation ReadoutMemory(
```

Comment:
This one needs to move to a utility place I think since its being used more broadly than before.

Reply:
totes agree!